tf_files/configs/kube-setup-sheepdog.sh

```diff
@@ -65,7 +65,7 @@ for user in sheepdog peregrine; do
       psql -t -U $gdcapi_db_user -h $gdcapi_db_host -d $gdcapi_db_database -c "CREATE USER $new_db_user WITH PASSWORD '$new_db_password';"
       psql -t -U $gdcapi_db_user -h $gdcapi_db_host -d $gdcapi_db_database -c "GRANT SELECT ON ALL TABLES IN SCHEMA public TO $new_db_user;"
```

Comment:
we should create a group user and add those users to a group, and then use [graph-grant](https://github.com/uc-cdis/datamodelutils/blob/master/datamodelutils/postgres_admin.py#L483) to grant the group user access. Also need to add the graph-grant to our database migration protocol ( didn't have that because the creator was the only user)